cmd/kops/rolling-update_cluster.go

```diff
@@ -189,7 +189,7 @@ func NewCmdRollingUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd.Flags().BoolVarP(&options.Yes, "yes", "y", options.Yes, "Perform rolling update immediately; without --yes rolling-update executes a dry-run")
 	cmd.Flags().BoolVar(&options.Force, "force", options.Force, "Force rolling update, even if no changes")
-	cmd.Flags().BoolVar(&options.CloudOnly, "cloudonly", options.CloudOnly, "Perform rolling update without confirming progress with Kubernetes")
+	cmd.Flags().BoolVar(&options.CloudOnly, "cloudonly", options.CloudOnly, "Perform rolling update without validating cluster status, which may cause downtime - only recommended for troubleshooting broken clusters")
```

Comment:
What do you think?
```suggestion
	cmd.Flags().BoolVar(&options.CloudOnly, "cloudonly", options.CloudOnly, "Perform rolling update without validating cluster status (will cause downtime)")
```

Reply:
Yeah, that's shorter so that's probably better for CLI output. I added a bit of explanatory text to the command description.

IMO it would be better to rename `--cloudonly` to `--skip-validation` since that is a more accurate description of what it does (and should also raise the right alarm bells in users' heads), but I'm not sure it's worth changing that at this point.